os_net_config/common.py

```diff
@@ -324,6 +326,60 @@ def is_vf(pci_address):
     return is_sriov_vf
 
 
+def get_pci_address(name):
+    """Fetch the PCI address of the interface
```

Comment:
I believe ethtool gets this info from the same place, so the results should be the same and it should be much faster to make a filesystem call than making an external process call to ethtool.